libs/ngx-mime/src/lib/core/access-keys-handler-service/access-keys.service.spec.ts

```diff
@@ -1,65 +1,48 @@
 import { HttpClientTestingModule } from '@angular/common/http/testing';
-import { inject, TestBed } from '@angular/core/testing';
-import { ContentSearchDialogConfigStrategyFactory } from '../../content-search-dialog/content-search-dialog-config-strategy-factory';
+import { TestBed } from '@angular/core/testing';
+import { provideAutoSpy } from 'jasmine-auto-spies';
 import { ContentSearchDialogService } from '../../content-search-dialog/content-search-dialog.service';
-import { InformationDialogConfigStrategyFactory } from '../../information-dialog/information-dialog-config-strategy-factory';
 import { InformationDialogService } from '../../information-dialog/information-dialog.service';
 import { SharedModule } from '../../shared/shared.module';
-import { AltoServiceStub } from '../../test/alto-service-stub';
-import { ViewDialogConfigStrategyFactory } from '../../view-dialog/view-dialog-config-strategy-factory';
 import { ViewDialogService } from '../../view-dialog/view-dialog.service';
 import { AltoService } from '../alto-service/alto.service';
 import { CanvasService } from '../canvas-service/canvas-service';
-import { ClickService } from '../click-service/click.service';
-import { FullscreenService } from '../fullscreen-service/fullscreen.service';
-import { HighlightService } from '../highlight-service/highlight.service';
 import { IiifContentSearchService } from '../iiif-content-search-service/iiif-content-search.service';
 import { IiifManifestService } from '../iiif-manifest-service/iiif-manifest-service';
-import { MimeViewerIntl } from '../intl';
 import { MimeDomHelper } from '../mime-dom-helper';
-import { MimeResizeService } from '../mime-resize-service/mime-resize.service';
 import { ModeService } from '../mode-service/mode.service';
 import { ContentSearchNavigationService } from '../navigation/content-search-navigation-service/content-search-navigation.service';
-import { StyleService } from '../style-service/style.service';
-import { ViewerLayoutService } from '../viewer-layout-service/viewer-layout-service';
 import { ViewerService } from '../viewer-service/viewer.service';
 import { AccessKeysService } from './access-keys.service';
 
 describe('AccessKeysService', () => {
+  let service: AccessKeysService;
+
   beforeEach(() => {
     TestBed.configureTestingModule({
       imports: [SharedModule, HttpClientTestingModule],
```

Comment:
You could remove HttpClientTestingModule as all other dependencies uses autospies